step.yml

```diff
@@ -127,54 +107,77 @@ inputs:
         **NOTE:** Only available from XCode 11.0 and above.
```

Comment:
Pls remove the NOTE: Xcode version >= 11 is required for App Store deployment.

Reply:
fixed